packages/story-editor/src/components/library/panes/media/local/mediaPane.js

```diff
@@ -355,7 +365,25 @@ function MediaPane(props) {
                 )}
               </SearchCount>
             )}
-            {!isSearching && (
+            {!isSearching && enableHotlinking && (
+              <ButtonsWrapper>
+                <LinkInsertion />
+                {hasUploadMediaAction && (
```

Comment:
Just noting that this check is new behavior here and ought to be handled by #8697. It caused an e2e test to fail which wasn't updated accordingly. I quickly fixed this in 76c8995a3d5f4a7ec446347bb9e7e31c87eb85f4 now.